modules/local/metabinner_kmer/main.nf

```diff
@@ -0,0 +1,36 @@
+process METABINNER_KMER {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/metabinner:1.4.4--hdfd78af_0' :
+        'quay.io/biocontainers/metabinner:1.4.4--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    val val_min_contig_size
+
+    output:
+    tuple val(meta), path("*_kmer_4_f${min_contig_size}.csv.gz"), emit: composition_profile
+    path "versions.yml"                                         , emit: versions
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    min_contig_size = val_min_contig_size ?: "1000"
+    def VERSION = '1.4.4-0' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
```

Comment:
I feel there is a big risk this line here will be forgotten when updating the module. Maybe move this comment close to the container declaration instead.

Same comments for all other local modules using metabinner.

Reply:
Good point. I thought I followed the recommendation in https://nf-co.re/docs/guidelines/components/modules#emission-of-versions, but I missed that there is indeed a second comment recommended close to the container.
Shall I make a PR to dev with these changes?